assets/src/components/cd/globalServices/GlobalServicesColumns.tsx

```diff
@@ -86,6 +91,27 @@ export const ColProject = columnHelper.accessor(
   }
 )
 
+export const ColMgmt = columnHelper.accessor(({ node }) => node?.mgmt, {
+  id: 'mgmt',
+  header: 'Mgmt',
+  cell: ({ getValue }) =>
+    getValue() ? (
```

Comment:
It will be included if the value is undefind as well (only excluded if explicitly false)

Reply:
Fixed the check.